examples/trials/mnist-nas/config.yml

```diff
@@ -0,0 +1,20 @@
+authorName: default
+experimentName: example_mnist
```

Comment:
could you add a config test for this example for the daily test?

Reply:
sure.